### DISCUSSION TITLE
Running faster on video clip input

### DISCUSSION THREAD

**Discussion by omer-re** [hacksider/Deep-Live-Cam#552] [Q&A]
While this great app can run on an online, live, streaming video input from the camera, when I try applying it on an existing mp4 video it takes forever (on the same env and HW).
I wasn't sure I how to set it so it'll treat the input video in a way that it will process it in a similar (even if not the same) speed as the live.
I assume it means some tradeoff for frame dropping, which I am cool with.

On a Macbook pro, 1 minute mp4 takes me 1.5 hours.

Thanks!

**Comment by hacksider** [comment]
It's doing it frame by frame. So if you want it to do live, you can use obs and use it's virtual camera and start from there.

**Comment by omer-re** [comment]
It feels like an unnecessary workaround, I can just change frame rate on
the code, on the ffmpeg part, I was just checking wether there's an
existing flag I can use

**Comment by hacksider** [comment]
That's just a suggestion. Also the fastest I can think of. If you want for it to be as fast as the live one, I suggest making it 720p or lower.

**Comment by omer-re** [comment]
I can reduce it to 10 fps for example, I'm just not sure if it won't still
try parse the video to the same fps, which will make my attempt inefficient
